packages/x-charts/src/themeAugmentation/props.d.ts

```diff
@@ -1,3 +1,4 @@
+import { BarLabelProps } from '../BarChart/BarLabel/BarLabel';
```

Comment:
This import looks slightly too verbose, could we re-export this type? 🤔

Reply:
Yup, I re-wrote the file structure and exported all the necessary BarLabel components and types